### ISSUE TITLE
Write SetFile class to load a file and expose its variables.

### ISSUE THREAD

**Issue by kenorb** [EA31337/EA31337-classes#7]
Write `SetFile` class (`SetFile.mqh`) and its methods with the following goals:

- method which can load any [`.set` file](https://github.com/EA31337/EA31337-Lite-Sets/blob/master/EURUSD/spread-10/digits-5/2014/EA31337-Lite.set) into class variables (e.g. `LoadFromFile(string file)`),
- another similar method to load from the string (e.g. `LoadFromText(string text)`)
- after loading the file, the class will parse and store individual variables (via its own setter method, e.g. `SetValue`) with its values into class instance (e.g. into array),
- write a getter method (e.g. `GetValue`) which expose the stored variables, so for given key will return its value,
- calling the setter on existing key will replace the value.

The code should work for both MQL4/MQL5 syntax.

--

In other words, the task expects one class with 4 methods. One to load the file (in `key=val` format, sample ini-like format linked) into class array variable, another load from text representation (they both actually can be combined), and one setter (to set) and getter method to load the value based on the key (from the loaded settings). The technical documentation is not needed, since I give freedom to how this may be implemented. I've added placeholder [class](https://github.com/EA31337/EA31337-classes/blob/master/SetFile.mqh) for a guidance.

--

Est. 6-8h

**Comment by OverloadedWolf** [comment]
You need documentation before anyone can truly be effective at aiding in this project.

**Comment by kenorb** [comment]
Sorry if that's not clear enough. The task expects one class with 4 methods. One to load the file (in `key=val` format, like ini, sample format linked) into class array, another load from text representation (they both actually can be combined), and one setter (to set) and getter method to load the value based on the key (from the loaded settings). The technical documentation is not needed, since I give freedom to how this may be implemented.

**Comment by kenorb** [comment]
I've added placeholder [class](SetFile.mqh) for a guidance. If I spent 20 minutes more on documentation, I'd quicker complete the task my-self.

**Comment by OverloadedWolf** [comment]
Okay, fair enough.

**Comment by lu43n** [comment]
Done
